### PR TITLE
New generated code and docs for getClusterMetrics

### PR DESCRIPTION
**Pull request by marians**
This updates the client to work with the latest spec.

Note that based on our tagging, the `getClusterMetrics` is added to two APIs: ClustersApi and MetricsApi.

### CODE REVIEW

**Comment by marians** [comment]
The tests fail and I have to first understand why.

File: test/model/V4NodeMetrics.spec.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if (typeof object[setter] === 'function')
+      object[setter](value);
+    else
+      object[property] = value;
+  }
+
+  describe('V4NodeMetrics', function() {
+    it('should create an instance of V4NodeMetrics', function() {
+      // uncomment below and update the code to test V4NodeMetrics
+      //var instane = new GiantSwarmV4.V4NodeMetrics();
```
Comment (oponder) [inline test/model/V4NodeMetrics.spec.js]: Is this generated code? what's the idea here.. Also theres a typo in all the commented examples.

Comment (marians) [reply test/model/V4NodeMetrics.spec.js]: It's all generated

Comment (marians) [reply test/model/V4NodeMetrics.spec.js]: I actually fixed that already https://github.com/swagger-api/swagger-codegen/commit/90859575ebfea800dde93a498e1439011963d6cf

**Comment by marians** [comment]
Covered by https://github.com/giantswarm/giantswarm-js-client/pull/48

**Pull request closed without merging**